src/components/Dashboard/EventCard.vue

```diff
@@ -114,6 +113,24 @@ const hasCall = computed(() => {
 		&& props.eventRoom.start * 1000 <= (Date.now() - 600_000) // 10 minutes buffer
 })
 
+const attachmentInfo = computed(() => {
+	if (hasAttachments.value) {
+		const file = Object.entries(props.eventRoom.eventAttachments)[0]?.[1]
+		const totalAttachments = Object.keys(props.eventRoom.eventAttachments).length
+
+		return {
+			icon: OC.MimeType.getIconUrl(file.fmttype) || imagePath('core', 'filetypes/file'),
+			label: file.filename,
```

Comment:
Does something like left-trim exist in JS?
```suggestion
			label: file.filename.ltrim('/'),
```

Reply:
I was assuming it will handled by API? if not, yeah, we can do it here